scripts/deploy-cursorless-nvim.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# This script is used to push to the cursorless.nvim github production repo
+set -euo pipefail
+
+# Clone current cursorless.nvim main
+mkdir -p dist && cd dist
+git clone 'https://github.com/hands-free-vim/cursorless.nvim.git' cursorless.nvim-remote
+cd -
+
+out_dir=dist/cursorless.nvim-remote
+
+# Delete the old files
+cd "$out_dir"
+git rm -r README.md lua/ vim/ node/
```

Comment:
why not just remove everything?

Reply:
You are right, we can delete everything too